ssl/statem/statem_srvr.c

```diff
@@ -785,6 +785,13 @@ WORK_STATE ossl_statem_server_pre_work(SSL_CONNECTION *s, WORK_STATE wst)
         return WORK_FINISHED_CONTINUE;
 
     case TLS_ST_SW_SESSION_TICKET:
+        if (SSL_CONNECTION_IS_DTLS(s)) {
+            /*
+             * We're into the last flight. We don't retransmit the last flight
+             * unless we need to, so we don't use the timer
+             */
+            st->use_timer = 0;
```

Comment:
Hmmm....this begs the question: what happens if after receiving the client's Finished we decided not to send any tickets (e.g. because num_tickets is 0)? In that case the timer will still be set won't it?

See:

https://github.com/openssl/openssl/blob/6a4a714045415be6720f4165c4d70a0ff229a26a/ssl/statem/statem_srvr.c#L556-L577

Reply:
It also suggests that we might need to do something extra to turn off the timer on the client side when it goes into its last flight for DTLSv1.3 because the client is always the last to transmit in DTLSv1.3

Reply:
DTLS 1.3 introduces a new concept of ACK records. I've been working on an implementation of such records. I believe that might take care of these uncertainties about correct use of the timer. So lets keep this PR out of review until I know more.